app/controllers/shelter_reviews_controller.rb

```diff
@@ -14,6 +14,17 @@ def create
     redirect_to "/shelters/#{shelter.id}"
   end
 
+  def edit
+    @shelter_review = ShelterReview.find(params[:id])
+    @shelter = Shelter.find(@shelter_review.shelter_id)
+    @shelter_review.update(shelter_review_params)
+    # redirect_to "/shelters/#{@shelter.id}"
```

Comment:
I commented this out because my errors hadn't lead me to this point yet and I didn't want to get ahead of myself.

Reply:
Thanks for letting me know.  I'll take a look at this once I've got a sense of how to approach this error.